### ISSUE TITLE
use docker image ID as value for version label

### ISSUE THREAD

**Issue by jonboulle** [appc/docker2aci#15]
Per OOB discussion, it would be helpful for people to correlate between Docker images and ACIs created through docker2aci by using the docker image's Image ID as the _version_ label in the produced ACI.

**Comment by philips** [comment]
I am thinking we should standardize some of these details from the conversion process. Here are my suggestions:

- `appc.io/docker/v1/checksum`
- `appc.io/docker/v1/tag`
- `appc.io/docker/v1/repository`
- `appc.io/docker/v1/imageID`
- `appc.io/docker/v1/parentID`

**Comment by iaguis** [comment]
Something like this makes total sense to me

**Comment by philips** [comment]
Seems like we should have a WELLKNOWN.md file with well-known labels or something.

**Comment by jonboulle** [comment]
We should loop in the quay.io guys here too, hmm?

**Comment by josephschorr** [comment]
We currently add:

```
{"name": "quay.io/derived-image", "value": synthetic_image_id},
```

where `synthetic_image_id` is the image ID we generate for the squashed image. This allows us to map any ACI we generate back to an image, and thereafter back to a repository and tag.

**Comment by iaguis** [comment]
Fixed by #78
